common/changes/office-ui-fabric-react/magellan-componentIdInternal_2018-02-06-19-27.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "[TextField, Panel] Removed componentId internal",
+      "type": "minor"
```

Comment:
I think this is a patch, if you leave the prop intact.

Reply:
Ok, done.